packages/AlgernonTests.package/ALGTestBackground.class/instance/testBackgroundIsAbandonedCorrectly.st

```diff
@@ -0,0 +1,7 @@
+testing
+testBackgroundIsAbandonedCorrectly
```

Comment:
Should we maybe move this test in an ALGTest Subclass ? I find the manual creation and cleanup of Algernon instances a bit "unpleasant".

Reply:
I don't think we should as this is a destructive use of the core. That would make reuse impossible.